protocols/manage/count.go

```diff
@@ -49,18 +58,23 @@ func NewCount(n *sda.Node) (sda.ProtocolInstance, error) {
 	return p, nil
 }
 
+func (p *ProtocolCount) Myself() string {
+	return fmt.Sprint(p.Entity().Addresses, p.Node.TokenID())
+}
+
 func (p *ProtocolCount) Dispatch() error {
 	for {
-		dbg.Lvl3("waiting for message in", p.Entity().Addresses)
+		dbg.Lvl3(p.Myself(), "waiting for message during", p.Timeout)
 		select {
 		case pc := <-p.PrepareCountChan:
-			dbg.Lvl3("Received from", pc.TreeNode.Entity.Addresses,
-				pc.TreeNode.Id)
+			dbg.Lvl3(p.Myself(), "received from", pc.TreeNode.Entity.Addresses,
+				pc.Timeout)
+			p.Timeout = pc.Timeout
 			p.FuncPC()
 		case c := <-p.CountChan:
 			p.FuncC(c)
```

Comment:
I know you like it short, but FuncC / FuncPC are *really* short ... what do they do ?